spec/models/ecf_participant_eligibility_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ECFParticipantEligibility, type: :model do
```

Comment:
I think I'd like to see a test for the `touch` on user a la https://github.com/DFE-Digital/early-careers-framework/blob/develop/spec/models/participant_profile_spec.rb#L16

Reply:
And the same for the validation data record

Reply:
Good idea